docker/Dockerfile

```
@@ -70,11 +69,14 @@ RUN \
     ldconfig && \
     # Install mongodb driver from source
     cd /opt && \
-    curl -kOL https://github.com/mongodb/mongo-cxx-driver/archive/legacy-1.1.2.tar.gz && \
-    tar xfz legacy-1.1.2.tar.gz && \
-    cd mongo-cxx-driver-legacy-1.1.2 && \
-    scons --use-sasl-client --ssl && \
-    scons install --prefix=/usr/local --use-sasl-client --ssl && \
+    wget https://github.com/mongodb/mongo-c-driver/releases/download/1.16.0/mongo-c-driver-1.16.0.tar.gz && \
+    tar xfvz mongo-c-driver-1.16.0.tar.gz && \
+    cd mongo-c-driver-1.16.0 && \
```

Let's use the newest at the present moment, which happens to be 1.17.4:

```suggestion
    tar xfvz mongo-c-driver-1.17.4.tar.gz && \
    cd mongo-c-driver-1.17.4 && \
```